.github/workflows/non-english-warning.yml

```diff
@@ -0,0 +1,45 @@
+name: Check for non-English updates outside of Crowdin
+
+on:
+  pull_request:
+    paths:
+      - "public/content/translations/**/*.md"
+      - "src/intl/**/*.json"
+      - "!src/intl/en/**
```

Comment:
Will check every PR for changes to these paths. If _any_ file in the diff includes:
- a markdown file within `public/content/translations` or
- a JSON file within `src/intl` (but NOT in `src/intl/en`)
the job will run.

Reply:
Only `.md` and `.json`; images and other files could be updated separately without Crowdin.